shell/platform/darwin/ios/framework/Source/FlutterEmbedderKeyResponder.mm

```diff
@@ -349,15 +349,15 @@ @interface FlutterEmbedderKeyResponder ()
  *
  * Set by the initializer.
  */
-@property(nonatomic) FlutterSendKeyEvent sendEvent;
+@property(nonatomic, copy, readonly) FlutterSendKeyEvent sendEvent;
 
 /**
  * A map of pressed keys.
  *
  * The keys of the dictionary are physical keys, while the values are the logical keys
  * of the key down event.
  */
-@property(nonatomic) NSMutableDictionary<NSNumber*, NSNumber*>* pressingRecords;
+@property(nonatomic, retain, readonly) NSMutableDictionary<NSNumber*, NSNumber*>* pressingRecords;
```

Comment:
Why do we have `readonly` private properties?

Reply:
I'm aware of the property vs ivar debate; I actually don't have a strong opinion on that. I was just curious about the addition of `readonly` to a private proprety. It just seems odd to say that code within a class can modify the collection, but not replace it with a different collection.

But this wasn't intended to be a PR-blocking question.

Reply:
> I'm aware of the property vs ivar debate; I actually don't have a strong opinion on that.
Guess I'm always geared up for a religious war in these kinds of reviews 😉 

Re: the readonly, you're right, that wasn't a static analyzer warning.  I did it because the property wasn't being set other than in `init`, so it could be made readonly, which I did in the spirit of preferring stricter property attributes, readonly, immutable, nonnull, etc.  But yeah, it's not API just private so it doesn't really matter (unless you're KVOing on the property and need to handle updating notifications in the property setter, which we aren't).  I can change it back if you prefer.